embedded/tbtree/tbtree.go

```diff
@@ -1193,10 +1193,6 @@ func (t *TBtree) SnapshotSince(ts uint64) (*Snapshot, error) {
 		return nil, ErrorToManyActiveSnapshots
 	}
 
-	if t.compacting {
```

Comment:
Why it had to be removed?

Reply:
to be able to load the catalog while compaction is in progress